### PR TITLE
[DFE-441] modify anvil create workspaces script for van allen lab

### PR DESCRIPTION
**Pull request by schaluva**
1. set bucket creation to us-central1
2. remove authorization domain steps
3. modify OWNER and READER access
4. fix tsv output to have only necessary columns (take out AD cols)
5. set default project to van allen project
6. create readme and utils.py (to be refactored)

Tested by creating two workspaces - success and fail modes.

### CODE REVIEW

File: scripts/van_allen_tools/set_up_vanallen_workspaces.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    """Make the json request to pass into create_workspace()."""
+
+    # initialize empty dictionary
+    create_ws_request = {}
+
+    create_ws_request["namespace"] = project
+    create_ws_request["name"] = workspace_name
+    create_ws_request["attributes"] = {}
+    create_ws_request["noWorkspaceOwner"] = False
+    create_ws_request["bucketLocation"] = "us-central1"
```
Comment (mmorgantaylor) [inline scripts/van_allen_tools/set_up_vanallen_workspaces.py:112]: this is always for sure going to be `us-central1` ? any benefit in having this be the default but allowing the user to input a different location if desired?

**Review by mmorgantaylor — APPROVED** [review]
one question about bucket location but otherwise lgtm!

File: scripts/van_allen_tools/set_up_vanallen_workspaces.py (hunk 1 above)
Comment (schaluva) [reply scripts/van_allen_tools/set_up_vanallen_workspaces.py:112]: I thought about making it user modifiable but I think the lab wanted to be super careful and have all their resources to be in that region so as not to incur accidental egress - so i thought a hard coded region might be best! i could see how somewhere done the line at some point *maybe* they might want to change regions? what do you think? just make it a variable and set default?

Comment (mmorgantaylor) [reply scripts/van_allen_tools/set_up_vanallen_workspaces.py:112]: ah hm that makes sense. maybe then make it super obvious by making a top-level global variable `TARGET_BUCKET_LOCATION = "us-central1"` and then here `create_ws_request["bucketLocation"] = TARGET_BUCKET_LOCATION`

Comment (schaluva) [reply scripts/van_allen_tools/set_up_vanallen_workspaces.py:114]: sweet - same thought. just updated

**Pull request merged**